### PR TITLE
Setup deployment

### PR DESCRIPTION
**Pull request by hochiw**
- Setup helm chart
- Created deployment templates and configs for react app, bdjuno and graphql server
- Added makefile for easier access to deployment commands
- Updated README.md on deployment information

- Tested by deploying onto oursky k8s 

based on #16
refs #11

### CODE REVIEW

File: deploy/likedao/templates/NOTES.txt [hunk 1]
```diff
@@ -1,22 +0,0 @@
-1. Get the application URL by running these commands:
```
Comment (chihimng) [inline deploy/likedao/templates/NOTES.txt]: please fixup this commit?

File: deploy/Makefile [hunk 2]
```diff
@@ -0,0 +1,24 @@
+BUILD_TAG ?= git-$(shell git rev-parse --short=8 HEAD)
+APP_NAME := likedao
+NAMESPACE ?=
+ENV ?=
```
Comment (chihimng) [inline deploy/Makefile:4]: is this used?

File: deploy/likedao/.gitignore [hunk 3]
```diff
@@ -1,3 +1,4 @@
 static/bdjuno.conf.yaml
 static/react-app.config.js
-static/react-app.nginx.conf
\ No newline at end of file
+static/react-app.nginx.
```
Comment (chihimng) [inline deploy/likedao/.gitignore]: missing conf extension?

File: deploy/likedao/values.sample.yaml [hunk 4]
```diff
@@ -0,0 +1,26 @@
+graphqlServer:
+  imageName: gcr.io/oursky-kube/likedao/likedao-graphql-server
```
Comment (chihimng) [inline deploy/likedao/values.sample.yaml:2]: ghcr?

File: deploy/likedao/.gitignore [hunk 5]
```diff
@@ -1,4 +1,4 @@
 static/bdjuno.conf.yaml
 static/react-app.config.js
-static/react-app.nginx.
+static/react-app.nginx.conf
```
Comment (chihimng) [inline deploy/likedao/.gitignore:3]: please fixup commit next time

**Review by chihimng — CHANGES_REQUESTED** [review]

**Comment by chihimng** [comment]
plz rebase 🙏

**Comment by hochiw** [comment]
@chihimng updated :adore2x:

File: README.md [hunk 6]
```diff
@@ -42,4 +42,29 @@ Please visit [here](./bdjuno/README.md#development) for more information
 
 ## Deployment
 
-TODO
+To execute a deployment for the LikeDAO system, please review the files in the [deployment folder](./deploy/) and follow the instructions below.
+
+## Prerequisite
+
```
Comment (chihimng) [inline README.md:48]: how about specs for db & steps to seed the initial schema?

**Review by chihimng — APPROVED** [review]

**Comment by chihimng** [comment]
@hochiw code lgtm, see if you want to handle docs now or track with issue

**Comment by hochiw** [comment]
Added an issue for this https://github.com/oursky/likedao/issues/39

**Pull request merged**